upgrade-ls

```diff
@@ -118,6 +118,13 @@ else
   echo
 fi
 
+# update installed wallet nodes
+
+echo
+echo "Updating installed wallet nodes."
+echo
+lamassu-update-wallet-nodes >> ${LOG_FILE} 2>&1
+
```

Comment:
I think this should happen before we show the message that the l-s update is complete, no?

Reply:
@josepfo not exactly what I mean, is just that we're updating the nodes even if the l-s update fails, this should be happening only after a successfull l-s update, hence the "before showing the L-S message"